packages/components/src/utils/src/types.ts

```diff
@@ -41,3 +41,19 @@ export interface RenderPropsHookOptions<T> extends RenderProps<T>, SharedDOMProp
 export interface BaseComponentProps extends DOMProps, AriaLabelingProps, SlotProps {
 
 }
+
+/** Added these for when we need to force an interactive state like for radios inside list box items */
+export interface InteractionProps {
+    /**
+     * @ignore
```

Comment:
Won't having `@ignore` here cause problems when generating documentation?

Reply:
The idea in orbiter is that those properties were only used for internal developement purpose, and we excluded them from the doc on purpose

Reply:
right, that's true, users shouldn't need to use them. This detail slipped my mind :P thanks!